### PR TITLE
Added 33 currencies in the list + minor corrections

### PR DESCRIPTION
**Pull request by nick1981**

### CODE REVIEW

File: english/src/in-which-countries-is-it-available.md [hunk 1]
```diff
@@ -52,21 +75,31 @@ Here we present you the list of currencies:
...
+| TWD  | New Taiwan Dollar           | 🇹🇼  |
+| TZS  | Tanzanian Shilling          | 🇹🇿  |
 | UAH  | Ukrainian Hryvnia           | 🇺🇦  |
+| UGX  | Ugandan Shilling            | 🇺🇬  |
 | USD  | US Dollar                   | 🇺🇸  |
 | UYU  | Peso uruguayo               | 🇺🇾  |
+| UZS  | Uzbekistan Som              | 🇺🇬  |
 | VES  | Bolívar                     | 🇻🇪  |
 | VND  | Vietnamese Dong             | 🇻🇳  |
+| XAF  | CFA Franc BEAC              | 🇨🇲🇨🇫🇨🇬🇬🇦🇬🇶🇹🇩  |
```
Comment (grunch) [inline english/src/in-which-countries-is-it-available.md:97]: XAF and XOF are being used both in Chad? acording with wikipedia they are using XAF

**Review by Catrya — COMMENTED** [review]
The number of currencies the bot accepts must also be changed. In the outdated version it says 58 but currently it is 94

**Review by Catrya — COMMENTED** [review]
Why change the currency of Uruguay for that of Ukraine?

File: french/src/how-do-i-create-a-community.md
Comment (Catrya) [inline french/src/how-do-i-create-a-community.md:1]: Why change the currency of Uruguay for that of Ukraine?

File: french/src/recommended-wallets.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 Les portefeuilles suivants sont ceux qui ont montré de meilleures performances lorsqu'ils ont été testés avec le bot :
 
+- [Wallet of Satoshi](https://www.walletofsatoshi.com/)
```
Comment (Catrya) [inline french/src/recommended-wallets.md:5]: It hasn't really performed that well with the bot, for example if you cancel a Sats sale it takes a few hours to update your wallet balance.

**Comment by nick1981** [comment]
> french/src/how-do-i-create-a-community.md
> 
> Why change the currency of Uruguay for that of Ukraine?

No special reason. Uruguay currency can stay there since it is only for documentation

File: english/src/in-which-countries-is-it-available.md (hunk 1 above)
Comment (nick1981) [reply english/src/in-which-countries-is-it-available.md:97]: That is correct. Tchad is using XAF.

File: french/src/recommended-wallets.md (hunk 2 above)
Comment (nick1981) [reply french/src/recommended-wallets.md:5]: ok thank you for the notice. but apart for cancellation delay, it is very speedy for normal transactions and also widely used. hope they will fix that in future versions...

**Comment by nick1981** [comment]
Hi @grunch can you process this request so that i can move forward with further contributions please?

**Review by grunch — APPROVED** [review]
LGTM! thanks

**Pull request merged**